### DISCUSSION TITLE
How to make all apps night mode/dark at once?

### DISCUSSION THREAD

**Discussion by prateekamana** [qtile/qtile#3495] [Q&A] [answered]
For example browsers and file managers. Just like desktop environments have a night mode theme option, does our twm also have such an option that we can change in config.py?

**Comment by SweepingBishops** [comment, marked as answer]
No, qtile doesn't do that. If you want that you'll have to change your gtk or qt config for apps that use them. Applications that don't use them should have their own configuration options. The best you can do without a desktop environment is to write a shell script that automatically changes everything (it'll just be easier to manually change the themes for the apps you use). That said if you have installed qtile alongside a DE, its settings should still work.

**Reply by prateekamana** [reply]
Manually changing them is what I have been doing, but I haven't been able to change the theme of nemo-file manager sadly, looked around but couldn't find useful info the last time I tried.

**Reply by m-col** [reply]
Check out `lxappearance`, it's a small GUI tool for configuring GTK themes, maybe that will affect nemo which seems to be GTK

**Reply by prateekamana** [reply]
That solution worked like a charm, thanks :D
